tests/__helpers__/schemas.ts

```diff
@@ -0,0 +1,117 @@
+export const validSchema: any = {
```

Comment:
This is from the MDT 👍  Felt like a good way to  start validating as we put our heart and soul into it!